lib/tree.rb

```diff
@@ -61,14 +68,13 @@ def tree_data
       diameter_sum += apple.diameter
     end
 
-    avg_diameter = # It's up to you to calculate the average diameter for this harvest.
+    avg_diameter = diameter_sum/basket.size
```

Comment:
Looks like this is doing integer division.  Is that what is intended?

Reply:
Yes, that's what I meant to do. Am I doing it incorrectly?

Reply:
Suppose you had basket with two apples in it, with diameters of 7 and 8, respectively.  What would you expect the average diameter to be?  What does this code say the average diameter is?